CONTRIBUTORS.yml

```diff
@@ -20,3 +20,4 @@
 - reaganrazon
 - jehron
 - nyccreator
+- donchuru
```

Comment:
It looks like your machine may be removing the final newline - would you mind enabling the `EditorConfig for VS Code` extension in your workspace so that it respects the `.editorconfig` configuration? And then resave/recommit this file?

Reply:
Sounds good. How about now?

Reply:
Perfect!